authentication/AuthenticationExampleSwift/MainViewController.swift

```diff
@@ -382,6 +398,7 @@ class MainViewController: UITableViewController, GIDSignInUIDelegate {
     }
     // [END auth_listener]
     self.microsoftProvider = OAuthProvider.init(providerID:"microsoft.com");
+    self.twitterProvider = OAuthProvider.init(providerID:"twitter.com");
```

Comment:
style nit: The `.init` before the parens is optional and should be omitted.

Reply:
Done.